src/lib/core/fiber_channel.h

```diff
@@ -207,6 +211,13 @@ fiber_channel_is_empty(struct fiber_channel *ch)
 	return ch->count == 0;
 }
 
+/**
+ * If fiber channel should work in a graceful close mode,
+ * i.e. become write-closed instead of closed until buffer
+ * is emitted by recipient. Is set by tarantool.compat.
+ */
+extern bool fiber_channel_graceful_close;
```

Comment:
I vote for getter and setter functions. Anyone up for it?

Reply:
It is purely internal, why encapsulate one more time?

Reply:
It is how we write the code in tarantool.

Reply:
Added `fiber_channel_set_graceful_close(bool is_graceful)`.